src/index.js

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
```

Comment:
Have less changes - there is no or minimal changes in this file.. but diff seems confusing.. if you are not changing anything in some file don't commit that.